packages/editor/src/utils/url.js

```diff
@@ -1,26 +1,8 @@
-/**
- * External dependencies
- */
-import { deburr, trim } from 'lodash';
-
 /**
  * WordPress dependencies
  */
-import { addQueryArgs } from '@wordpress/url';
-
-/**
- * Returns the URL of a WPAdmin Page.
- *
- * TODO: This should be moved to a module less specific to the editor.
- *
- * @param {string} page  Page to navigate to.
- * @param {Object} query Query Args.
- *
- * @return {string} WPAdmin URL.
- */
-export function getWPAdminURL( page, query ) {
-	return addQueryArgs( page, query );
```

Comment:
this was exported from the `@wordpress/editor` package so I wonder if we need to deprecate it as well and leave it in for now.

Reply:
ah very good. I saw `* from 'utils'` and got that wrong